cypress/common/i-can-enter-use-information/generic.spec.ts

```diff
@@ -36,3 +38,24 @@ export const iCanEditMyNUses = (n: number): void => {
   iCanSeeAButtonContaining("Continue");
   iCanSeeAButtonContaining("Add another");
 };
+
+export const whenIGoToEditTheUseNumber = (useNumber: number) => {
+  iHaveClickedOnAGivenLink(
+    `${CreateRegistrationPageURLs.environment}?useIndex=${useNumber}`
+  );
+  whenIClickContinue();
+};
+
+export const iAmOnTheLandBranchForUseNumber = (useNumber: number) => {
```

Comment:
Hey @SianRMadeTech these functions are throwing linter warnings about a missing return type.  You just need to... 
```suggestion
export const iAmOnTheLandBranchForUseNumber = (useNumber: number): void => {
```